Inc/DDSTextureLoader.h

```diff
@@ -99,6 +99,7 @@ namespace DirectX
         _In_ unsigned int cpuAccessFlags,
         _In_ unsigned int miscFlags,
         _In_ bool forceSRGB,
```

Comment:
Since this is already a *breaking change*, I'd recommend replacing ``bool forceSRGB`` with a [DEFINE_ENUM_FLAG_OPERATORS](https://walbourn.github.io/modern-c++-bitmask-types/) like I have for other functions--I haven't done this one yet because it only has one option at the moment.

```
    enum DDS_LOADER_FLAGS : uint32_t
    {
        DDS_LOADER_DEFAULT = 0,
        DDS_LOADER_FORCE_SRGB = 0x1,
        DDS_LOADER_IGNORE_SRGB = 0x2,
        DDS_LOADER_SRGB_DEFAULT = 0x4,
    };

...

    HRESULT __cdecl CreateDDSTextureFromFileEx(
        _In_ ID3D11Device* d3dDevice,
        _In_z_ const wchar_t* szFileName,
        _In_ size_t maxsize,
        _In_ D3D11_USAGE usage,
        _In_ unsigned int bindFlags,
        _In_ unsigned int cpuAccessFlags,
        _In_ unsigned int miscFlags,
        _In_ DDS_LOADER_FLAGS loadFlags,
        _Outptr_opt_ ID3D11Resource** texture,
        _Outptr_opt_ ID3D11ShaderResourceView** textureView,
        _Out_opt_ DDS_ALPHA_MODE* alphaMode = nullptr) noexcept;

...

#ifdef __clang__
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-dynamic-exception-spec"
#endif

    DEFINE_ENUM_FLAG_OPERATORS(DDS_LOADER_FLAGS);

#ifdef __clang__
#pragma clang diagnostic pop
#endif
```

See WICTextureLoader [.h](https://github.com/microsoft/DirectXTK/blob/main/Inc/WICTextureLoader.h) / [.cpp](https://github.com/microsoft/DirectXTK/blob/main/Src/WICTextureLoader.cpp) as well as *DirectX Tool Kit for DX12* DDSTextureLoader [.h](https://github.com/microsoft/DirectXTK12/blob/main/Inc/DDSTextureLoader.h) / [.cpp](https://github.com/microsoft/DirectXTK12/blob/main/Src/DDSTextureLoader.cpp)

Reply:
I was thinking the same thing; I treaded a little too lightly in trying to be unintrusive...flags are the way to go for clarity and future extensibility.  I'll have those mods up shortly.